invokeai/app/services/shared/sqlite/sqlite_database.py

```diff
@@ -30,15 +33,20 @@ def __init__(self, config: InvokeAIAppConfig, logger: Logger):
 
         self.conn.execute("PRAGMA foreign_keys = ON;")
 
+        migrator = SQLiteMigrator(conn=self.conn, database=self.database, lock=self.lock, logger=self._logger)
```

Comment:
I'd be inclined to move the migration logic out of `SqliteDatabase`. I think of `SqliteDatabase` more as a DB connection, and I think it's more natural to tie the migration to app startup rather than `SqliteDatabase` construction.